modules/gdnative/nativescript/nativescript.cpp

```diff
@@ -119,20 +119,24 @@ String NativeScript::get_class_name() const {
 
 void NativeScript::set_library(Ref<GDNativeLibrary> p_library) {
 	if (!library.is_null()) {
-		WARN_PRINT("library on NativeScript already set. Do nothing.");
+		WARN_PRINT("Library in NativeScript already set. Do nothing.");
 		return;
 	}
 	library = p_library;
-	lib_path = library->get_current_library_path();
+	if (library.is_valid()) {
```

Comment:
Shouldn't we do
```
if (p_library.is_null()) {
    return;
}
```
before assigning an empty library to `library` instead?

Reply:
That makes sense...